practice/src/index.html

```diff
@@ -12,149 +18,161 @@
+                <option value="">Z to A</option>
+              </optgroup>
+              <optgroup label="Price">
+                <option value="">Low to high</option>
+                <option value="">High to low</option>
+              </optgroup>
+            </select>
           </div>
         </div>
-      </form>
-    </div>
-
-    <!-- Sidebar -->
-    <aside class="sidebar">
-      <div class="sidebar-header">
-        <a href="javascript:void(0)" class="sidebar-logo">
-          <img src="./assets/images/logo.png" alt="logo" />
-        </a>
-      </div>
-      <div class="sidebar-body">
-        <ul class="nav-list">
-          <li class="nav-item active">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use xlink:href="./assets/images/icons/icons.svg#home-icon" />
+        <main class="homepage-main">
+          <!-- Card Add -->
+          <div class="card card-add-product">
+            <a href="javascrip:void(0)" class="card-add-title">
+              <svg width="15" height="15">
+                <use xlink:href="./assets/images/icons/icons.svg#add-icon" />
               </svg>
+              Add new dish
             </a>
-          </li>
-          <li class="nav-item">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use
-                  xlink:href="./assets/images/icons/icons.svg#discount-icon"
-                />
-              </svg>
-            </a>
-          </li>
-          <li class="nav-item">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use xlink:href="./assets/images/icons/icons.svg#graph-icon" />
-              </svg>
-            </a>
-          </li>
-          <li class="nav-item">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use
-                  xlink:href="./assets/images/icons/icons.svg#message-icon"
-                />
-              </svg>
-            </a>
-          </li>
-          <li class="nav-item">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use
-                  xlink:href="./assets/images/icons/icons.svg#notification-icon"
-                />
+          </div>
+          <!-- Card Products-->
+          <div class="card card-product">
+            <div class="card-header">
+              <svg width="15" height="15">
+                <use xlink:href="./assets/images/icons/icons.svg#delete-icon" />
               </svg>
-            </a>
-          </li>
-          <li class="nav-item">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use
-                  xlink:href="./assets/images/icons/icons.svg#setting-icon"
+            </div>
+            <div>
+              <figure class="card-image">
+                <img
+                  src="#"
```

Comment:
You should add an image URL here. When you use the API to render it, you can remove it later